plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -408,6 +430,16 @@ fu_dell_dock_ec_get_dock_info (FuDevice *device,
 	}
 	fu_device_set_version_lowest (device, self->ec_minimum_version);
 
+
+	/* TODO: Drop if minimum EC is set to 23+
+	 * Determine if the passive flow should be used when flashing
+	 */
+	if (fu_device_has_custom_flag (device, "try-passive-flow") &&
```

Comment:
maybe drop the `-try`?

Reply:
Well it is "try" because if you use use EC22 it won't activate.

Reply:
Sure.  I'll add this to the README